test/jasmine/tests/contour_test.js

```diff
@@ -648,4 +655,29 @@ describe('contour hover', function() {
             expect(pt).toEqual(undefined);
         });
     });
+
+    describe('for xyz-column traces', function() {
+        beforeAll(function(done) {
+            gd = createGraphDiv();
+
+            Plotly.plot(gd, [{
+                type: 'contour',
+                x: [1, 2, 3],
+                y: [2, 3, 4],
+                z: [10, 4, 20],
+                text: ['a', 'b', 'c'],
+                hoverinfo: 'text'
+            }])
+            .then(done);
+        });
+
+        afterAll(destroyGraphDiv);
+
+        it('should find closest point and should', function() {
```

Comment:
Something is missing after "and should"?

Reply:
...

https://github.com/plotly/plotly.js/blob/771e279736ae095aa8d7ff11a9170d76cdd779ec/test/jasmine/tests/contour_test.js#L679

---

When a test fails, this reads as "should find closest point and should have correct index".